web/app/scripts/recent-proportions/recent-proportions.html

```diff
@@ -0,0 +1,17 @@
+<div class="tab-content">
+    <div role="tabpanel" class="tab-pane active" id="insight01">
+        <h1><b>{{ recent.proportions[0].proportion }}</b> of all traffic accidents involved <b>{{ recent.proportions[0].label }} last week</b>.</h1>
```

Comment:
I realize it's just a placeholder, but might as well replace `traffic accidents` with `{{ ctl.recordType.plural_label | lowercase }}`.